app/Http/Livewire/AdminAreaDashboard.php

```diff
@@ -158,21 +158,31 @@ private function prepareLastSixMonthChart()
             ->withoutLegend()
             ->setDataLabelsEnabled(true);
 
-        $j = 5;
-        for ($i = 0; $i <= 5; $i++) {
-            $month = Carbon::now()->subMonths($j);
+            $year = Carbon::now()->year;
+            
+            /* mein alter Code ergänzt um die Year-Abfrage */
+            for ($i = 0; $i <= 5; $i++) {
+                $this->lastSixMonthChart->addSeriesPoint('Posts', date('M', mktime(null, null, null, $i)), Post::whereYear('created_at','=',$year)->whereMonth('created_at','=', $i)->count());
```

Comment:
Hier passt doch auch was nicht oder? Das Datum wird über die Zählvariable abgefragt. Also 1 bis 5. Es gibt doch mehr Monate als 1-5 
```php
->whereMonth('created_at','=', $i)
```

Reply:
Uuuuh. Yepp... Nehme den Approve vielleicht lieber zurück... :-D

Reply:
0 bis 5. das sind 6 Monate, oder?

Reply:
Ja, aber das Problem ist, dass das nur zufällig funktioniert, weil wir grade Mai haben. Die Loop sagt: Gib mir erst alles von 2021 im Monat 0, dann von 2021 im Monat 1, dann von 2021 im Monat 2, ... Wenn wir grade November hätten, würde das nicht mehr klappen.

Reply:
Und für den Dezember passt's schon nicht, weil wir hier nach dem Dezember in 2021 fragen und nicht in 2020. Dates und Timezones... immer wieder lustig. https://www.youtube.com/watch?v=-5wpm-gesOY

Reply:
whereMonth(0) dürfte auch nicht funktionieren